src/app/functions/server/checkSession.ts

```diff
@@ -18,6 +18,9 @@ export function checkSession(session: Session | null) {
   } else if (!session.user.subscriber.id) {
     logger.warn("no_subscriber_id_in_session", { session });
     return false;
+  } else if (session.error === "RefreshAccessTokenError") {
+    console.error("Refreshing access token failed... require login again");
```

Comment:
Why `console` and not `logger` here? I think structured logging is better in general in newer code unless we can't for some reason (e.g. it doesn't support JS using it directly)

Reply:
Copy pasted, will fix